cynic/Cargo.toml

```diff
@@ -23,6 +23,7 @@ serde_json = "1.0"
 cynic-proc-macros = { path = "../cynic-proc-macros", version = "0.9.0" }
 chrono = { version = "0.4.11", optional = true}
 bson = { version = "1.1.0", optional = true}
+uuid = { version = "0.8.1", default-features = false, optional = true}
```

Comment:
Why the `default-features = false`?  Looks like the default feature just activates `std` support, so not sure we need to disable (though happy to go with it if there's a reason I'm unware of...)

Reply:
I've missed that. There is no reason to disable default features (std in our case). I'll update PR